backend/Controllers/SightingController.cs

```diff
@@ -27,20 +37,57 @@ public ActionResult<ListResponse<Sighting>> GetSightingsBySpeciesId([FromRoute]
         {
             var sightings = _sightings.GetSightingsBySpeciesId(speciesId);
             return new ListResponse<Sighting>(sightings);
-        }   
+        }
 
-        [HttpGet("/pending")] 
-        public ActionResult<ListResponse<Sighting>>GetPendingSightings()
+        [HttpGet("/pending")]
+        public ActionResult<ListResponse<Sighting>> GetPendingSightings()
         {
             var pendingSightings = _sightings.GetPendingSightings();
             return new ListResponse<Sighting>(pendingSightings);
-        }   
+        }
 
         [HttpPost]
         public ActionResult CreateSighting([FromBody] CreateSightingRequest createSightingRequest)
-        {   
+        {
             var createdSighting = _sightings.CreateSighting(createSightingRequest);
             return Created("/api", createdSighting);
         }
+
+        [HttpPatch]
+        public ActionResult ConfirmOrRejectSighting([FromHeader] string authorization, [FromBody] ConfirmOrRejectRequest confirmOrRejectRequest)
+        {
+            if (authorization is null)
+            {
+                return new UnauthorizedResult();
+            }
+
+            try
+            {
+                var encodedUsernamePassword = authorization.Substring("Basic ".Length).Trim();
+                string usernamePassword = AuthHelper.Base64Decode(encodedUsernamePassword);
+                int separatorIndex = usernamePassword.IndexOf(':');
```

Comment:
You don't need this line! The `Split(':')` below finds the colon itself, so you don't need to search for it here.